components/clearbutton/index.css

```diff
@@ -112,6 +119,12 @@
 	}
 }
 
+.spectrum-ClearButton:disabled {
```

Comment:
I know there's a few other components that double up the selector here like this: 

```
.spectrum-ClearButton:disabled,
.spectrum-ClearButton.is-disabled
```

I'm curious if you think we need that here or not. 🤷‍♀️ I'm not sure I have a strong preference either way, just thinking.

Reply:
I was debating whether to add that since it behaves without it, but I dropped it in for the sake of consistency. ✨